Demo/Demo/Models/NativeCheckoutResult.swift

```diff
@@ -1,8 +0,0 @@
-import Foundation
-import PaymentsCore
-
-enum NativeCheckoutResult {
```

Comment:
If we're removing this result type, what type are we returning to the merchant?

Reply:
this is done with callbacks from nxo previously we were encapsulating result